lib/device/library/linux.rb

```diff
@@ -17,5 +17,56 @@ def get_device_root_dir(volume_name)
       end
       nil
     end
+
+    # 端末のデバイスファイル名と uhelper オプションの値を取得
+    def device_mount_info(volume_name)
+      device_root = get_device_root_dir(volume_name)
+      raise Device::CantEject, "端末が接続されていません" if device_root.nil?
+
+      pattern = %r!^(/dev/[^ ]+) .* #{device_root} .*\Wuhelper=(\w+)!
+      open("|mount") do |f|
+        while line = f.gets
+          if line =~ pattern
+            return [$1, $2]
+          end
+        end
+      end
+      [nil, nil]
+    end
+
+    def ejectable?(volume_name)
+      begin
+        case device_mount_info(volume_name)[1]
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not 4) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->